docs/docs/reference/other-new-features/export.md

```diff
@@ -90,10 +90,10 @@ Export clauses can appear in classes or they can appear at the top-level. An exp
 
 ### Motivation
 
-It is a standard recommendation to prefer aggregation over inheritance. This is really an application of the principle of least power: Aggregation treats components as blackboxes whereas inheritance can affect the internal workings of components through overriding. Sometimes the close coupling implied by inheritance is the best solution for a problem, but where this is not necessary the looser coupling of aggregation is better.
+It is a standard recommendation to prefer composition over inheritance. This is really an application of the principle of least power: Aggregation treats components as blackboxes whereas inheritance can affect the internal workings of components through overriding. Sometimes the close coupling implied by inheritance is the best solution for a problem, but where this is not necessary the looser coupling of composition is better.
```

Comment:
There is still one reference to "Aggregation": `... of least power: Aggregation treats ...`